tests/benchmarks/CompilerServiceBenchmarks/Benchmarks.fs

```diff
@@ -0,0 +1,381 @@
+﻿namespace FSharp.Compiler.Benchmarks
+
+open System
+open System.IO
+open System.Text
+open System.Threading.Tasks
+open FSharp.Compiler.CodeAnalysis
+open FSharp.Compiler.Diagnostics
+open FSharp.Compiler.EditorServices
+open FSharp.Compiler.Text
+open FSharp.Compiler.AbstractIL.IL
+open FSharp.Compiler.AbstractIL.ILBinaryReader
+open BenchmarkDotNet.Attributes
+open BenchmarkDotNet.Running
+open FSharp.Compiler.Benchmarks
+
+[<AutoOpen>]
+module BenchmarkHelpers =
+
+    type Async with
```

Comment:
Should we reuse it from the test utilities?
https://github.com/dotnet/fsharp/blob/fca1cd7bb08f01aa7abd34876d6ee748ec5fc8e9/tests/FSharp.Test.Utilities/Utilities.fs#L21